build/ci/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_the_containers_are_runninig=true
+  for pod in $pods; do
+    running_containers=`echo $pod | awk -F / '{print$1}'`
+    wanted_containers=`echo $pod | awk -F / '{print$2}'`
```

Comment:
`running_containers_count`
`total_containers_count`

Reply:
done

Reply:
I don't see it

Reply:
now you will see it, I mistakenly didn't do it as a review comments